front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/ItineraryMarkers.tsx

```diff
@@ -57,9 +57,9 @@ const formatPointWithNoName = (
   </>
 );
 
-const extractMarkerInformation = (pathSteps: (PathStep | null)[], showStdcmAssets: boolean) =>
-  pathSteps.reduce((acc, cur, index) => {
-    if (cur && cur.coordinates) {
+const extractMarkerInformation = (pathSteps: (PathStep | null)[], showStdcmAssets: boolean) => {
+  return pathSteps.reduce((acc, cur, index) => {
```

Comment:
You can remove the return here and the brackets ;)

Reply:
done